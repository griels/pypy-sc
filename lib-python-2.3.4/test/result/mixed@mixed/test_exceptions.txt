command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_exceptions.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 00:13:40 2005
pypy-revision: 11549
OUTPUT TEST
see output in: /u/pedronis/PyPy/dist/lib-python-2.3.4/test/result/test_exceptions.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
==========================timeout==========================
Traceback (application-level):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_exceptions.py", line 95 in ?
    while 1: x = x+x
KeyboardInterrupt
FAILED: test output differs
========================== closed ==========================
execution time: 900.337150097 seconds
exit status: 1
