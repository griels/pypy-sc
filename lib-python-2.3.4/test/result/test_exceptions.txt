command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_exceptions.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 01:58:14 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_exceptions.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
==========================timeout==========================
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_exceptions.py", line 95 in ?
    while 1: x = x+x
KeyboardInterrupt
========================== closed ==========================
execution time: 900.142353058 seconds
exit status: 1
