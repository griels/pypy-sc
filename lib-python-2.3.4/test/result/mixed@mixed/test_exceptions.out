test_exceptions
5. Built-in exceptions
spam
<pypy type 'AttributeError'>
spam
<pypy type 'EOFError'>
errno=None args=('spam',) strerror=None filename=None
<pypy type 'IOError'>
spam
<pypy type 'ImportError'>
spam
<pypy type 'IndexError'>
'spam'
<pypy type 'KeyError'>
spam
<pypy type 'KeyboardInterrupt'>
(not testable in a script)
spam
<pypy type 'MemoryError'>
(not safe to test)
spam
<pypy type 'NameError'>
spam
<pypy type 'OverflowError'>
